docs/user-guide/faq.md

```diff
@@ -13,6 +13,12 @@ Please see README.md in the for the release by clicking on the tag for the relea
 ### How do I run the setup on my laptop?
 Follow the [guide on the wiki](https://github.com/vmware/docker-volume-vsphere/wiki/Using-laptop-for-running-the-entire-stack)
 
+### How do I bind a volume to a particular container host?
```

Comment:
Here, by "bind" I mean that a volume is closely tied to a container host, and not in a literal "attach" sense. Do you suggest that we use "attach" still?